src/app.typedefs.ts

```diff
@@ -0,0 +1,34 @@
+export interface User {
+  id: number;
+  name: string;
+  username: string;
+  email: string;
+  address: {
+    street: string;
+    suite: string;
+    city: string;
+    zipcode: string;
+    geo: {
+      lat: string;
+      lng: string;
+    };
+  };
```

Comment:
readme hint advises that you could've type only the required fields you'll be working with,
but if  you decided to full-scale, better use standalone interfaces for nested types:
```suggestion
  address: Address;
  ...
}
    
interface Address {
  street: string;
  suite: string;
  city: string;
  zipcode: string;
  geo: Geo;
}

interface Geo {
  lat: string;
  lng: string;
}   
```